design/spaces.go

```diff
@@ -23,6 +23,7 @@ var genericLinksForSpace = a.Type("GenericLinksForSpace", func() {
 	a.Attribute("related", d.String)
 	a.Attribute("backlog", d.String, `URL to the backlog work items`)
 	a.Attribute("meta", a.HashOf(d.String, d.Any))
+	a.Attribute("workItemTypes", d.String, "URL to list all WITs for this space")
```

Comment:
done in https://github.com/almighty/almighty-core/pull/1033/commits/a10c79a1a04b8618f4cde4fa216b44f6b663f225